src/main/resources/db/changelog/v1.0/V1_001_create-student-table.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+
+    <include file="/db/changelog/v1.0/shared-properties.xml"/>
+
+    <changeSet id="create-student-table" author="polinatonko">
+
+        <createTable tableName="student">
+            <column name="id" type="uuid" defaultValueComputed="${uuid_function}">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="first_name" type="varchar(30)"/>
+            <column name="last_name" type="varchar(30)"/>
+            <column name="email" type="varchar(254)">
+                <constraints unique="true" nullable="false"/>
+            </column>
+            <column name="date_of_birth" type="date"/>
+            <column name="coins" type="number" defaultValueNumeric="0">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+    </changeSet>
```

Comment:
as I mentioned above, it's better include all other changesets with tables creation inside of one changelog file related to current application version